src/components/process/manage/ManageProcess.js

```diff
@@ -51,24 +83,26 @@ export const ManageProcess = () => {
                             <Grid item><Typography variant="h5">Recruitment Process Settings</Typography></Grid>
                         </Grid>
                         <Grid item> <Divider/> </Grid>
-                        <Grid container style={{display: "flex", flexGrow: 1}}>
-                            {/*<Grid item>*/}
-                            {/*  Beginning of recruitment process in future maybe  */}
-                            {/*</Grid>*/}
-                            <Grid item>
-                                <ManageEndDate selectedDate={selectedEndDate || new Date()}
-                                               onChange={(date) => setSelectedEndDate(date)}
-                                               processStartDate={process?.startDate}/>
-                            </Grid>
-                        </Grid>
+                        <div style={{display: 'inline-block'}}>
+                            <div style={{ float: 'left', marginLeft: '4em', marginTop: '1em' }}>
+                                <Button variant="contained" color="primary" size="large" onClick={startProcess} disabled={isProcessStarted()}>Start Process</Button>
+                            </div>
+                            <div style={{ float: 'right', marginRight: '4em' }}>
+                                <ManageDate selectedDate={selectedEndDate}
+                                                onChange={(date) => setSelectedEndDate(date)}
+                                                processStartDate={process?.startDate}
+                                />
+                            </div>
+                        </div>
+
                         {/*<Grid item>*/}
                         {/*    In future tasks and quizzes? */}
                         {/*</Grid>*/}
```

Comment:
Hmm to nie jest widok z perspektywy hr'a? Bo jeśli tak to tu nie powinno być ani tasków ani quizów widać co najwyżej notatki

Reply:
Tak to jeszcze było zanim wymyśliliśmy jak ogarnąć deva, ale komentarz zostawiłem, żeby potem było wiadomo, gdzie to dawać XDD